instrumentation/instamongo/v2/version.go

```diff
@@ -0,0 +1,6 @@
+// (c) Copyright IBM Corp. 2025
+
+package instamongo
+
+// Version is the instrumentation module semantic version
+const Version = "1.28.1"
```

Comment:
```suggestion
const Version = "1.0.0"
```

We can then release the major version, and the release script will update the version to 2.0.0.